Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11
```

Comment:
I'm not sure, but I found [this *old* comment](https://github.com/orgs/python-poetry/discussions/1879#discussioncomment-216865) that suggests a completely different (and seem to be more robust) approach to building the Docker image.

@nilnil47